Automattic/newspack-plugin
# Add Button Component

### All Submissions:


### Changes proposed in this Pull Request:

Adds a component providing a `NewspackButton`, inheriting largely from the default `Button` element but overridden with Muriel styling. Includes styling for the various [props available](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/button#props).

Closes #19 .

### How to test the changes in this Pull Request:

1. Apply the patch
2. In WP Admin, navigate to Newspack > Subscriptions
3. You will see a pink "Continue" button
4. In development, change the `isPrimary` prop of `<NewspackButton>` within [wizards/subscriptions/index.js](https://github.com/Automattic/newspack-plugin/blob/3ff96ef964c5b07a43f2a20b5bdd5b898ee317a5/assets/src/wizards/subscriptions/index.js#L38) to one of the [other values](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/button#props) to see the different styles.

### Other information:
